AddqBittorrentTrackers.sh

```diff
@@ -309,7 +319,12 @@ elif [ $auto_tor_grab -eq 0 ]; then # manual run
 		done < <(echo $torrent_list | $jq_executable --raw-output '.[] | .hash')
 	else
 		for i in "${tor_arg_names[@]}"; do
-			torrent_name_list=$(echo "$torrent_list" | $jq_executable --raw-output --arg tosearch "$i" '.[] | select(.name | ascii_downcase | contains($tosearch | ascii_downcase)) .name') #possible fix for ONIGURUMA regex libary
+
+            if [ $category_instead -eq 1 ]; then
+			    torrent_name_list=$(echo "$torrent_list" | $jq_executable --raw-output --arg tosearch "$i" '.[] | select(if (.save_path | endswith("complete")) then "uncategorized" else .save_path end | ascii_downcase | contains($tosearch | ascii_downcase)) .name') 
```

Comment:
This doesn't seem to be tied to all categories, just uncategorized or full downloads, it seems more like custom work for a need of yours, though maybe I'm wrong.
What is your idea?